src/clib/xtg/surf_export_irap_ascii.c

```diff
@@ -85,8 +85,8 @@ surf_export_irap_ascii(FILE *fc,
         fcode = 2;
     }
 
-    xmax = xori + (mx - 1) * xinc;
-    ymax = yori + (my - 1) * yinc;
+    xmax = xori + mx * xinc;
+    ymax = yori + my * yinc;
 
```

Comment:
No, maps are node oriented, so correct lengh is (NCOL-1) * XINC. This is verified as same calculation as in RMS

![image](https://user-images.githubusercontent.com/16700455/116547011-9d53d100-a8f2-11eb-8dec-1ea34d91e00c.png)

Reply:
Are you sure that is correct? I have reverted it, and that makes this fail: https://github.com/equinor/xtgeo/pull/523/checks?check_run_id=2492053842

My test is that I write a surface and then read it and check that the result is the same as what I originally created.

Reply:
Fails with: 
```
{'xinc': 0.6666666666666666, 'yinc': 0.6666666666666666} !=  ('xinc': 2.0'yinc': 2.0})
```
where 2.0 is what I used to create the surface with.